uploader.py

```diff
@@ -27,21 +28,33 @@
 except FileNotFoundError:
     sys.exit("No settings found")
 
+enable_email = config["Email"]["enabled"].lower() in ['true'] #I hate python sometimes...
```

Comment:
The Eamail > enabled section in config must be exists, otherwise exception will be thrown. The check will only work with strings. In the [Documentation](https://docs.python.org/3/library/configparser.html#supported-datatypes) you will find a better option, [getboolean()](https://docs.python.org/3/library/configparser.html#configparser.ConfigParser.getboolean)

Reply:
Derp, I knew I was forgetting something.  Thanks for pointing out a better way!